### PR TITLE
DOC-4367 RDI K8s docs

### PR DESCRIPTION
**Pull request by andy-stark-redis**
[DOC-4367](https://redislabs.atlassian.net/browse/DOC-4367)

I'll make a separate PR to add the release notes to the RDI repo when we're happy with them.

[DOC-4367]: https://redislabs.atlassian.net/browse/DOC-4367?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/integrate/redis-data-integration/release-notes/rdi-1-4-0.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+RDI keeps the Redis cache up to date with changes in the primary database, using a [_Change Data Capture (CDC)_](https://en.wikipedia.org/wiki/Change_data_capture) mechanism.
+It also lets you _transform_ the data from relational tables into convenient and fast data structures that match your app's requirements. You specify the transformations using a configuration system, so no coding is required.
+
+## Headlines
+
+-   Installation on [Kubernetes]({{< relref "/integrate/redis-data-integration/installation/install-k8s" >}}) using a [Helm chart](https://helm.sh/docs/). You can install on [OpenShift](https://docs.openshift.com/) or other flavours of K8s using Helm.
+
+- Improvements for installation on VMs:
+	- Installer checks if the OS firewall is enabled on Ubuntu and RHEL.
```
Comment (yaronp68) [inline content/integrate/redis-data-integration/release-notes/rdi-1-4-0.md:30]: What is it linked to?

File: content/integrate/redis-data-integration/release-notes/rdi-1-4-0.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	- Installer verifies DNS resolution from RDI components.
+	- Installer provides log lines from components that failed during RDI deployment if a problem occurs.
+	- Improved verification of RDI installation.
+	- Installer verifies if the RDI database is in use by another instance of RDI.
+	- Installer checks and warns if any [`iptables`](https://www.netfilter.org/projects/iptables/index.html) rules are set.
+	- Improved message when RDI tries to connect to its Redis database with invalid TLS keys.
```
Comment (yaronp68) [inline content/integrate/redis-data-integration/release-notes/rdi-1-4-0.md:36]: what is it linked to?

File: content/integrate/redis-data-integration/release-notes/rdi-1-4-0.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Limitations
+
+- RDI can write data to a Redis Active-Active database. However, it doesn't support writing data to two or more Active-Active replicas. Writing data from RDI to several Active-Active replicas could easily harm data integrity as RDI is not synchronous with the source database commits.
+- RDI write-behind (which is currently in preview) should not be used on the same data set that RDI ingest is writing to Redis. This would either cause an infinite loop or would harm the data integrity, since both ingest and write-behind are asynchronous, eventually-consistent processes.
```
Comment (yaronp68) [inline content/integrate/redis-data-integration/release-notes/rdi-1-4-0.md:41]: Remove write-behind. We already removed it to another section in docs and about to stop selling it in few days

File: content/integrate/redis-data-integration/architecture.md [hunk 4]
```diff
@@ -136,14 +136,28 @@ traffic. The diagram below shows this configuration:
...
 ### RDI on Kubernetes
 
-You can run RDI in a namespace on your own Kubernetes cluster with
-a separate deployment for each of the control plane
-components (REST API, operator, and metrics exporter). The operator
-creates and configures deployments for the collector and stream processor
-when you start a pipeline from the CLI tool. This means that you must
-provide the operator with a service account that has permissions to create and manipulate pods in the namespace. 
+You can use the RDI [Helm chart](https://helm.sh/docs/topics/charts/) to install
+on [Kubernetes (K8s)](https://kubernetes.io/). This creates:
```
Comment (yaronp68) [inline content/integrate/redis-data-integration/architecture.md:145]: it's good to explicitly mention Openshift as most customers will use it
something like on Kubernetes (K8s) including Red Hat Openshift

**Review by yaronp68 — CHANGES_REQUESTED** [review]

File: content/integrate/redis-data-integration/release-notes/rdi-1-4-0.md (hunk 1 above)
Comment (andy-stark-redis) [reply content/integrate/redis-data-integration/release-notes/rdi-1-4-0.md:30]: The links are to the Helm docs and the OpenShift docs. Would you prefer to remove the links?

File: content/integrate/redis-data-integration/release-notes/rdi-1-4-0.md (hunk 2 above)
Comment (andy-stark-redis) [reply content/integrate/redis-data-integration/release-notes/rdi-1-4-0.md:36]: Links to the iptable home page. Again, I'll remove it if you think it's better without it.

File: content/integrate/redis-data-integration/release-notes/rdi-1-4-0.md (hunk 1 above)
Comment (yaronp68) [reply content/integrate/redis-data-integration/release-notes/rdi-1-4-0.md:30]: I think they need to linking just the particular words or not have them at all

File: content/integrate/redis-data-integration/release-notes/rdi-1-4-0.md (hunk 2 above)
Comment (yaronp68) [reply content/integrate/redis-data-integration/release-notes/rdi-1-4-0.md:36]: the link needs to be just on the word IPtables

**Review by yaronp68 — CHANGES_REQUESTED** [review]

**Review by yaronp68 — APPROVED** [review]

File: content/integrate/redis-data-integration/installation/install-k8s.md [hunk 5]
```diff
@@ -0,0 +1,472 @@
...
+You should then provide the details of this database in the [`values.yaml`](#the-valuesyaml-file)
+file as described below.
+
+### Using a private image registry
+
+Add the RDI images from [Docker Hub](https://hub.docker.com/) to your local registry.
+The example below shows how to specify the registry and image pull secret in the
+[`values.yaml`](#the-valuesyaml-file) file for the Helm chart:
+
+```yaml
```
Comment (galilev) [inline content/integrate/redis-data-integration/installation/install-k8s.md:74]: @spicy-sauce do I remember correctly that you told me that you had tried the second format and it didn't work?

File: content/integrate/redis-data-integration/installation/install-k8s.md [hunk 6]
```diff
@@ -0,0 +1,472 @@
...
+    ```
+
+1.  Open the `values.yaml` file and set the appropriate values for your installation
+    (see [The `values.yaml` file](#the-valuesyaml-file) below for the full set of
+    available values).
+
+1.  Start the installation:
+
+    ```bash
+    helm install <The logical chart name> rdi-k8s/<rdi-tag>/helm
```
Comment (galilev) [inline content/integrate/redis-data-integration/installation/install-k8s.md:110]: please add --create-namespace -n rdi at the end of the helm install command

Comment (andy-stark-redis) [reply content/integrate/redis-data-integration/installation/install-k8s.md]: Done.

File: content/integrate/redis-data-integration/installation/install-k8s.md [hunk 7]
```diff
@@ -0,0 +1,472 @@
...
+
+Add the RDI images from [Docker Hub](https://hub.docker.com/) to your local registry.
+The example below shows how to specify the registry and image pull secret in the
+[`values.yaml`](#the-valuesyaml-file) file for the Helm chart:
+
+```yaml
+global:
+ imagePullSecrets: []
+ # - name: "image-pull-secret"
+ # or
```
Comment (galilev) [inline content/integrate/redis-data-integration/installation/install-k8s.md:78]: remove lines 78-79. Should be : 
```
 imagePullSecrets: []
 # - name: "image-pull-secret"
```

Comment (andy-stark-redis) [reply content/integrate/redis-data-integration/installation/install-k8s.md]: Done.

**Review by galilev — APPROVED** [review]

**Pull request merged**